src/transformers/models/auto/tokenization_auto.py

```diff
@@ -140,6 +140,7 @@
             ("layoutlmv3", ("LayoutLMv3Tokenizer", "LayoutLMv3TokenizerFast" if is_tokenizers_available() else None)),
             ("layoutxlm", ("LayoutXLMTokenizer", "LayoutXLMTokenizerFast" if is_tokenizers_available() else None)),
             ("led", ("LEDTokenizer", "LEDTokenizerFast" if is_tokenizers_available() else None)),
+            ("lilt", ("LayoutLMv3Tokenizer", "LayoutLMv3TokenizerFast" if is_tokenizers_available() else None)),
```

Comment:
Looking at the doc, there should also be a default processor for the auto-mapping, and probably a default feature extractor as well? (using LayoutLMv3 both time).

Reply:
This model actually doesn't require a feature extractor or processor, as the model doesn't take an image modality as input (unlike LayoutLMv3). The model only has a text modality as input, hence it only requires a tokenizer.

Reply:
You should update the doc above then, as it mentions `AutoProcessor`?

Reply:
Oh yes indeed, thanks for flagging!